eng/templates/Azure.ResourceManager.Template/src/autorest.md

```diff
@@ -16,6 +16,7 @@ sample-gen:
 skip-csproj: true
 modelerfour:
   flatten-payloads: false
+use-model-reader-writer: true
```

Comment:
@ArcturusZhang When are we planning to remove this flag to enable public serialization by default?

Reply:
I think default false is good.

Reply:
I think the final goal is to enable this be default, and we can simplify the code quite a bit.

Reply:
Yeah the plan is when we have majority of the libraries enabled on this feature, we will remove this flag, and making it default on.

Reply:
Why do we need to wait for majority of the SDKs to be enabled? 
We are working on the enabling of this for all MPG RPs.
If we remove it now, we don't need to add the flag to autorest.md.